bigip/resource_bigip_ltm_monitor.go

```diff
@@ -179,6 +185,10 @@ func resourceBigipLtmMonitorRead(d *schema.ResourceData, meta interface{}) error
 			if err := d.Set("destination", m.Destination); err != nil {
 				return fmt.Errorf("[DEBUG] Error saving Destination to state for Monitor (%s): %s", d.Id(), err)
 			}
+			if err := d.Set("compatibility", m.Compatibility); err != nil {
```

Comment:
This is a `TypeString` attribute, it doesn't require error checking (that is done by the underlying terraform schema code)